apps/rush-lib/assets/rush-init/rush.json

```diff
@@ -61,7 +61,21 @@
      * After modifying this field, it's recommended to run "rush update --full" so that the package manager
      * will recalculate all version selections.
      */
-    /*[LINE "HYPOTHETICAL"]*/ "resolutionStrategy": "fast"
+    /*[LINE "HYPOTHETICAL"]*/ "resolutionStrategy": "fast",
+
+    /**
+     * This is the mode for tracking external dependencies to detect if a project needs to be rebuilt
+     * during an incremental build.
+     *
+     * If set to "project," track individual projects' dependencies when detecting if a project needs
+     * to be rebuilt during an incremental build.
+     *
+     * If set to "repository," rebuild all projects when any dependencies across the repository are changed
```

Comment:
This seems ambiguous to me, "when any dependencies across the repository are changed during an incremental build". It is just me, or could this be better defined?

Reply:
Yeah I wasn't thrilled with that phrasing either. Care to take a stab at it?

Reply:
```
If set to "repository," all projects will be rebuilt if any external dependencies anywhere in the repository are changed.
```